jscomp/bsb/bsb_ninja_rule.ml

```diff
@@ -170,16 +170,19 @@ let make_custom_rules
     );
     if has_ppx then 
       Buffer.add_string buf " $ppx_flags"; 
+    if gen_simple then
```

Comment:
could gen_simple be made a ref too so it does not need be passed down

Reply:
Done!